Changelogs for Digital Twin Manager Plugins 1.3.2 (#2730)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020-1-21-asset-properties-widget-2-0-displays-different-list-of-properties-then-selected.md

---
date: ""

> title: "Asset Properties Widget 2.0 displays different list of properties then selected"

Reviewer: ```suggestion
title: "Asset Properties Widget now correctly displays the selected key value pairs of complex properties."
```

> title: "Asset properties widget now correctly displays the selected key value pairs of complex properties

Reviewer 2: ```suggestion
title: "Asset properties widget now correctly displays the selected key-value pairs of complex properties
```

> title: "Asset Properties Widget now correctly displays the selected key value pairs of complex properties."

Reviewer 2: ```suggestion
title: "Asset properties widget now correctly displays the selected key value pairs of complex properties
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1688"
version: "1020.1.21"
---

> Asset Properties Widget 2.0 displays different list of properties then selected

Reviewer 2: Please make this a proper release notes entry (context, issue, fix, impact).

> Asset Properties Widget 2.0 displays different list of properties then selected

Reviewer: ```suggestion
In the "Asset properties" widget in dtm-plugins, an issue was identified that resulted in displaying different complex property key-value pairs than those selected during widget configuration. This issue has been fixed.
```
